peachjam/resources.py

```diff
@@ -674,3 +673,12 @@ def before_save_instance(self, instance, using_transactions, dry_run):
             instance.password = make_password(instance.password)
             instance.is_staff = True
             instance.save()
+
+
+class AttorneyResource(resources.ModelResource):
+    class Meta:
+        model = Attorney
+        skip_unchanged = True
+        report_skipped = True
+        exclude = ("id", "description")
```

Comment:
I don't think we need to exclude these fields,

Reply:
What's the impact of `report_skipped` and `skip_unchanged`? We don't use those settings elsewhere (at least not explicitly), and we can safely both create and update?